recipes/configs/phi3/mini_full.yaml

```diff
@@ -41,21 +41,20 @@ checkpointer:
     model-00002-of-00002.safetensors
   ]
   recipe_checkpoint: null
-  output_dir: /tmp/Phi-3-mini-4k-instruct
+  output_dir: /tmp/finetuned-phi3-4k-model
   model_type: PHI3_MINI
 resume_from_checkpoint: False
 
 # Fine-tuning arguments
 batch_size: 2
-epochs: 3
+epochs: 1
```

Comment:
We only need one epoch here for POC

Reply:
Do we need a similar BS change here?

Reply:
No b/c I updated the ``gradient_accumulation_steps``